code/operators/solution/operators.sol.cpp

```diff
@@ -0,0 +1,196 @@
+#include <cassert>
+#include <iomanip>
+#include <iostream>
+#include <numeric>
+
+class Fraction {
+ public:
+  explicit Fraction(int i) : m_num(i), m_denom(1) {}
+  Fraction(int num, int denom) : m_num(num), m_denom(denom) {}
+
+  int num() const { return m_num; }
+  int denom() const { return m_denom; }
+
+  void normalize() {
+    const int gcd = std::gcd(m_num, m_denom);
+    m_num /= gcd;
+    m_denom /= gcd;
+  }
+
+  Fraction& operator*=(int i) {
+    m_num *= i;
```

Comment:
```suggestion
    m_num *= i;
    normalize();
```

Reply:
Thx! I also added a new `cout` to check this case as well.